packages/conduit-rxjs/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.6.0
+
+Published TBD
+
+1. Updated dependencies to latest version.
```

Comment:
Add this item. Even though `conduit-rxjs` doesn't use any operators, I'm fine leaving in the second sentence, so that there's an explicit reason for the change.

```md
2. Increased minimum peer dependency version of [RxJS to v7.2](https://github.com/ReactiveX/rxjs/blob/master/CHANGELOG.md#720-2021-07-05). Operators are now imported from the top-level `rxjs` module, since importing from `rxjs/operators` will be depreciated soon.
```